tests/test_workers/test_tasks/test_build.py

```diff
@@ -260,6 +260,29 @@ def test_push_image(mock_si, mock_run_cmd, mock_glps, schema_version):
         mock_run_cmd.assert_called_once()
 
 
+@mock.patch('iib.workers.tasks.build.run_cmd')
+def test_fix_schema_version_fail_retry_success(mock_run_cmd):
+
+    mock_run_cmd.side_effect = [IIBError('Something went wrong'), None]
+    destination = 'some_destination'
+    build._fix_schema_version(destination)
+
+    skopeo_args = mock_run_cmd.mock_calls[1][1][0]
```

Comment:
This line and the one below can be removed since this is already tested elsewhere. You just need to test that `run_cmd` was called twice to test the retry.

Reply:
Yup, but now that we have separate tests, the right place for these lines would be here right?